packages/quantic/cypress/e2e/default-1/tab/tab.cypress.ts

```diff
@@ -80,9 +80,11 @@ describe('quantic-tab', () => {
         });
 
         it('should work as expected', () => {
-          Expect.search.constantExpressionEqual(
-            tabs.case.expression,
-            param.useCase
+          Expect.sendNewSearchRequest(
+            param.useCase === 'search' ? 'interfaceLoad' : 'searchboxSubmit',
```

Comment:
Why is the actionCause `interfaceLoad` or `searchboxSubmit` for the tab tests?

Reply:
I guess `interfaceLoad` works when the page loads, but where is the `searchboxSubmit` coming from?

Reply:
`searchboxSubmit`  is here because of the insight use case.

In the insight use case, contrary to the search use case, the first search is not executed automatically. A specific logic is required to fetch the configuration of the insight panel and then set up all the components accordingly to then be able to execute the first search and initialize the insight panel correctly, for example this job is all handled by the InsightRender component in the C4SF package.

In the Quantic E2E tests for the insight use case and to avoid having to do all this logic just for the tests, we taking a shortcut by submit a manual search  using the search box, to initialize the insight panel correctly, so that's why the event that will be logged will be `searchboxSubmit`.

PS: this logic to initialize an insight panel using a search with the search box was not added in this PR, it started existing since we first started testing the insight use case.

Reply:
ahhhh ok I wasn't aware of the shortcut of submitting the manual search once.
All good then